packages/components/src/color-palette/index.tsx

```diff
@@ -228,7 +228,7 @@ function UnforwardedColorPalette(
 		__experimentalIsRenderedInSidebar = false,
 		...otherProps
 	} = props;
-	const clearColor = useCallback( () => onChange( undefined ), [ onChange ] );
+	const clearColor = useCallback( () => onChange( '' ), [ onChange ] );
```

Comment:
This may break third-party component users that on the onChange handler may have logic relying on the onChange parameter being undefined. I think it would be better to say the type is string | undefined.

Reply:
That's a good point! I'll leave it as-is and update types elsewhere accordingly, thank you!